react/components/SummaryItem.tsx

```diff
@@ -19,6 +24,10 @@ defineMessages({
     id: 'store/checkout-summary.Discounts',
     defaultMessage: 'Discounts',
   },
+  Taxa: {
```

Comment:
Shouldn't be `Tax`?
```suggestion
  Tax: {
```

Reply:
It should! At least now I know DefineMessages isn't being used :p